src/viam/components/arm/arm.py

```diff
@@ -45,13 +45,10 @@ async def move_to_position(
     ):
         """
         Move the end of the arm to the Pose specified in ``pose``.
-        When obstacles are specified in ``world_state``, the motion plan of the arm will avoid them.
 
         Args:
 
-            pose (Pose): The destination Pose for the arm.
-
-            world_state (WorldState): The obstacles for the arm to avoid on its way to ``pose``.
+            pose (viam.proto.common.Pose): The destination Pose for the arm.
```

Comment:
(q) This feels a little wordy. If necessary than disregard, but are we not able to keep this as just `Pose` (or analagous types below)?

Reply:
The reason for this change is that Sphinx is not able to find the same class `Pose` as the one linked in the header. (Problems already known in Sphinx with using multiple targets.) The description on the PR has an image that shows you what I mean.
We believe it's better to have the description point to the same link, even if it means it's wordier.

If anything, I would argue that we should be calling `from viam.components.arm import Pose` instead of `from viam.proto.common import Pose`. That way, both the header and comments match up without the wordiness, but I'm not entirely sure why this was done this way.

Reply:
Thank you for the feedback! It helped me take a step back and realized there might be a better way, which is exactly what I suggested by importing `Pose` from `arm` rather than `proto`. I'll be making those changes now!